Day3/Functions.md

```diff
@@ -174,9 +178,9 @@ if __name__ == "__main__":
 ```
 
 ### Exercise
-1. Move the calls to the temperature function into a __main__ section. Leave the def part of the function outside the if statement block. Run this program.
+1. Move the calls to the temperature function into a `__main__` section. Leave the def part of the function outside the if statement block. Run this program.
 
-2. Create a new program with just an if __name__ == "__main__": block. Put different code in this main section. Some suggestions:
+2. Create a new program with just an `if __name__ == "__main__"`: block. Put different code in this main section. Some suggestions:
```

Comment:
Perhaps the colon should be in the quotes?

Reply:
Good catch! Fixed it.